src/checks/quote_character.rs

```diff
@@ -16,7 +16,7 @@ impl Default for QuoteCharacterChecker<'_> {
     fn default() -> Self {
         Self {
             name: "QuoteCharacter",
-            template: "The value is wrapped in quotes",
+            template: "The value has quote characters (\', \") in it",
```

Comment:
For me, `in it` seems superfluous.
```suggestion
            template: "The value has quote characters (\', \")",
```